src/Symfony/Component/Mime/composer.json

```diff
@@ -17,6 +17,7 @@
     ],
     "require": {
         "php": ">=8.1",
+        "pear/crypt_gpg": "^1.6",
```

Comment:
This needs to be moved to `require-dev` as said in [previous comment](https://github.com/symfony/symfony/pull/50222#discussion_r1184761607).

Reply:
Haven't I moved it to require-dev in d5b9214a7832b152cbdb27b9c7b68bdfc07cc0df?
I'm glad you like the idea of that feature. A new component would be essential a standalone package under a certain namespace?

Reply:
> Haven't I moved it to require-dev in [d5b9214](https://github.com/symfony/symfony/commit/d5b9214a7832b152cbdb27b9c7b68bdfc07cc0df)? 

The commit sorts packages alphabetically, but does not move the package to `require-dev`.

Reply:
Yeah, I did that too. But it is actually under `require-dev` now:

https://github.com/symfony/symfony/blob/d5b9214a7832b152cbdb27b9c7b68bdfc07cc0df/composer.json#L123C1-L162

Reply:
https://github.com/symfony/symfony/commit/d5b9214a7832b152cbdb27b9c7b68bdfc07cc0df fixes the `composer.json` in the root dir, this is `src/Symfony/Component/Mime/composer.json`.

Reply:
I see. If it should't live in `symfony/mime` then it should be entirely removed from there and instead moved into a new component (for example `symfony/pgp`) where the dependency is handled?

Reply:
Hey,

the last months I tested my implementation in a Symfony app and discovered several flaws. For example it doesn't work with templated emails and sending via mail provider via api is also not possible due to the nature of a PGP structured mail.
I still would like to see that feature in Symfony and I'll work on that in the upcoming weeks.